tests/pytests/test_language.py

```diff
@@ -0,0 +1,144 @@
+    env.cmd('FT.DROP', 'idx')
+
+
+def testSearchJsonIndexLanguage(env):
+    conn = getConnectionByEnv(env)
+
+    # Create index - language Italian by default
+    env.cmd('FT.CREATE', 'idx', 'ON', 'JSON',
+            'LANGUAGE', 'italian', 'SCHEMA', '$.word', 'AS', 'word', 'TEXT')
+    
+    # Create sample data in Italian
+    conn.execute_command('JSON.SET', '{word}:1', '$', '{"word":"arancia"}') # orange
+    conn.execute_command('JSON.SET', '{word}:2', '$', '{"word":"arance"}') # oranges
+
+    # Create sample data in English
+    conn.execute_command('JSON.SET', '{word}:3', '$', '{"word":"orange"}') # orange
+    conn.execute_command('JSON.SET', '{word}:4', '$', '{"word":"oranges"}') # oranges
+
+    # Wait for index to be created
+    waitForIndex(env, 'idx')
+
+    # Search for "arancia", should use the language by default: Italian
+    # It should return 2 results using stemming in Italian
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'SORTBY', 'word', 'DESC', 
+                  'NOCONTENT')
+    env.assertEqual(res, [2, '{word}:1', '{word}:2'])
+
+    # Search for "arancia" using English
+    # It should return 1 results using invalid stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'language', 'english')
+    env.assertEqual(res, [1, '{word}:1', ['$', '{"word":"arancia"}']])
+
+    # Search for "oranges" using English
+    # It should return 2 results using stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'english',
+                  'SORTBY', 'word', 'ASC', 'NOCONTENT')
+    env.assertEqual(res, [2, '{word}:3', '{word}:4'])
+
+    # Search for "orange" using Italian
+    # It should return 1 results using invalid stemming in Italian
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'italian')
+    env.assertEqual(res, [1, '{word}:4', ['$', '{"word":"oranges"}']])
+
+    # drop index
+    env.cmd('FT.DROP', 'idx')
+
+def testSearchIndexLanguageField(env):
+    conn = getConnectionByEnv(env)
+    
+    # Create sample data in Italian
+    conn.execute_command('HSET', '{word}:1', 'word', 'arancia',
+                         '__lang', 'italian') # orange
+    conn.execute_command('HSET', '{word}:2', 'word', 'arance',
+                         '__lang', 'italian') # oranges
+    conn.execute_command('HSET', '{word}:3', 'word', 'mela',
+                         '__lang', 'italian') # apple
+    conn.execute_command('HSET', '{word}:4', 'word', 'mele',
+                         '__lang', 'italian') # apples
+    conn.execute_command('HSET', '{word}:5', 'word', 'orange',
+                         '__lang', 'english')
+    conn.execute_command('HSET', '{word}:6', 'word', 'oranges',
+                         '__lang', 'english')
+
+    # Create index - language Italian by default
+    env.cmd('FT.CREATE', 'idx', 'ON', 'HASH', 'PREFIX', '1', '{word}:',
+            'LANGUAGE', 'italian', 'LANGUAGE_FIELD', '__lang',
+            'SCHEMA', 'word', 'TEXT', '__lang', 'TEXT')
```

Comment:
Why do you need the last two argument here (`__lang`, `TEXT`)?

Reply:
You are right, I don'n need it. I removed it.